packs/spell-effects/aura-bane.json

```diff
@@ -0,0 +1,58 @@
+{
+    "_id": "F5NnpKkn4pKwEeFB",
+    "img": "systems/pf2e/icons/spells/bane.webp",
+    "name": "Aura: Bane",
+    "system": {
+        "badge": {
+            "labels": [
+                "10 ft.",
+                "20 ft.",
+                "30 ft.",
+                "40 ft.",
+                "50 ft.",
+                "60 ft.",
+                "70 ft.",
+                "80 ft.",
+                "90 ft.",
+                "100 ft."
+            ],
+            "loop": false,
+            "type": "counter",
+            "value": 1
+        },
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bane].</p><hr /><p>You fill the minds of your enemies with doubt. Enemies in the area must succeed at a Will save or take a –1 status penalty to attack rolls as long as they are in the area. Once per round on subsequent turns, you can Sustain the spell to increase the emanation's radius by 10 feet and force enemies in the area that weren't yet affected to attempt another saving throw.</p><hr /><p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Bane]</p>"
```

Comment:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Bane]</p><hr /><p>Enemies in the area must succeed at a Will save or take a –1 status penalty to attack rolls as long as they are in the area.</p><hr /><p>@UUID[Compendium.pf2e.spell-effects.Item.Spell Effect: Bane]</p>"
```

Reply:
I was trying to keep flavour consistnet between both bane and bless.

The Effect for Bane and bless essentially cuts all flavour so I cut what remained of Bless's effect flavour.

The Aura for Bless retains the spells flavour description so I maintained that for the Banes Aura for consistency.

Reply:
We want to strip out flavor on any effect. They should just say exactly what they automate so it's easier to parse for users. They can click the link to the source to read the full text.